server/src/http_server/templates/header.rs

```diff
@@ -45,14 +45,6 @@ impl Render for OpenGraph {
 }
 
 pub fn head(og: impl Borrow<OpenGraph>) -> Markup {
-    let temporary_remove_service_worker_script = r"
-      navigator.serviceWorker.getRegistrations().then(function(registrations) {
-        for(let registration of registrations) {
-            registration.unregister();
-        } 
-      });
```

Comment:
Its been a full year since my old blog that registered this service worker

Hopefully this was enough time for anyone with it still registered to visit and get it removed 🤞